src/imputation/tmi_imputation.py

```diff
@@ -563,6 +563,12 @@ def run_tmi(
     imputed_only_df = imputed_only_df.sort_values(
         ["formtype", "imp_class"], ascending=True
     ).reset_index(drop=True)
+    
+    # Set dtype of manual_trim column to bool before concatination
+    convert_dict = {'manual_trim': bool}
+    full_qa_df = full_qa_df.astype(convert_dict)
+    imputed_only_df = imputed_only_df.astype(convert_dict)
```

Comment:
I've done some analyis with copilot and I think all these cols are bools and are in those 2 dataframes:

```suggestion
    bool_like_cols =  ['manual_trim', 'empty_pgsic_group', 'empty_pg_group', '211_trim', '305_trim']
```

Reply:
do you think manual_trim is a special case?

Reply:
yes, I think that's best.